From b009d7b3a91d6d4908e43b0b69ca6b600c360f4b Mon Sep 17 00:00:00 2001
From: joshua Watt <JPEWhacker@gmail.com>
Date: Wed, 1 May 2024 16:21:58 -0600
Subject: [PATCH 21/22] classes/create-spdx-2.2: Fix SPDX Namespace Prefix

According to the SPDX documentation, it should be "spdxdocs" not
"spdxdoc"

[YOCTO #15398]

(From OE-Core rev: 9abcb18014020804738dfc7d278d7097679f4d19)

Signed-off-by: Joshua Watt <JPEWhacker@gmail.com>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>
(cherry picked from commit d1b25413ced62dc2927dae57b8d16e67d15dc220)
Signed-off-by: Steve Sakoman <steve@sakoman.com>
---
 meta/classes/create-spdx-2.2.bbclass | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/meta/classes/create-spdx-2.2.bbclass b/meta/classes/create-spdx-2.2.bbclass
index e6f274b87e..a3efc92aeb 100644
--- a/meta/classes/create-spdx-2.2.bbclass
+++ b/meta/classes/create-spdx-2.2.bbclass
@@ -28,7 +28,7 @@ SPDX_ARCHIVE_SOURCES ??= "0"
 SPDX_ARCHIVE_PACKAGED ??= "0"
 
 SPDX_UUID_NAMESPACE ??= "sbom.openembedded.org"
-SPDX_NAMESPACE_PREFIX ??= "http://spdx.org/spdxdoc"
+SPDX_NAMESPACE_PREFIX ??= "http://spdx.org/spdxdocs"
 SPDX_PRETTY ??= "0"
 
 SPDX_LICENSES ??= "${COREBASE}/meta/files/spdx-licenses.json"
